BentoKit.podspec

```diff
@@ -14,9 +14,10 @@ Pod::Spec.new do |s|
   s.ios.deployment_target = '10.0'
 
 # Switch back to use the version tag when we release 0.4.
-  s.source        = { :git => "https://github.com/Babylonpartners/Bento.git", :branch => "develop" } #, :tag => "#{s.version}" }
-  s.source_files  = 'BentoKit/BentoKit/**/*.swift'
-  s.exclude_files = 'BentoKit/BentoKit/Components.playground/**/*'
+  s.source           = { :git => "https://github.com/Babylonpartners/Bento.git", :branch => "develop" } #, :tag => "#{s.version}" }
```

Comment:
I do not think we have `develop` branch

Reply:
This was already there, but I'll change it to `master`.